driver.py

```diff
@@ -496,20 +496,21 @@ def generate_driver(p: Panel, options: Options) -> None:
 }}
 
 {simple.generate_mode(p)}
-static int {p.short_id}_get_modes(struct drm_panel *panel)
+static int {p.short_id}_get_modes(struct drm_panel *panel,
+			struct drm_connector *connector)
```

Comment:
I'm not really happy with the indentation here because it's currently "hardcoded" but as the previous line changes with the length of `p.short_id` it's not perfect.

Reply:
You can use

```python
{wrap.join(f'static int {p.short_id}_get_modes(', ',', ');', ['struct drm_panel *panel', 'struct drm_connector *connector'])}
```

to wrap it properly.

Basically it calculates the length of the prefix, and then appends the listed arguments, wrapping the line with the correct indent when necessary.

Reply:
:+1: